wifiphisher/pywifiphisher.py

```diff
@@ -1024,7 +1050,10 @@ def run():
     clients_APs = []
     APs = []
     args.accesspoint = ap_mac
-    args.channel = channel
+
+    if not (args.apname and args.channel):
+        args.channel = channel
```

Comment:
The only way to use `check_args` is to this:
```python
def check_args(args):
    if args.apname and args.channel:
        global_varialbe = True
```
And then check if `global_variable`is `True` instead of the current code.